src/settings-ui/Settings.UI/SettingsXAML/App.xaml.cs

```diff
@@ -208,17 +209,16 @@ protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs ar
             }
             else
             {
+#if DEBUG
                 // For debugging purposes
                 // Window is also needed to show MessageDialog
                 settingsWindow = new MainWindow(isDark);
                 settingsWindow.ExtendsContentIntoTitleBar = true;
                 settingsWindow.Activate();
                 settingsWindow.NavigateToSection(StartupPage);
-
-#if !DEBUG
-                ShowMessageDialogAndExit("The application cannot be run as a standalone process. Please start the application through the runner.", "Forbidden");
+                ShowMessageDialog("The application is running in Debug mode.", "DEBUG");
 #else
-                ShowMessageDialog("The application cannot be run as a standalone process. Please start the application through the runner.", "Forbidden");
+                SettingsDeepLink.OpenSettings(SettingsDeepLink.SettingsWindow.Overview, true);
 #endif
```

Comment:
```suggestion
                Exit();
#endif
```

Prevent the first instance of `Powertoys.Settings.exe` to stay opened after starting PT runner.
![image](https://github.com/microsoft/PowerToys/assets/25966642/503a1735-cc77-460a-95ee-cbfbb1c09acb)

Reply:
good catch. thanks